### ISSUE TITLE
MultiSelect Widget Questions

### ISSUE THREAD

**Issue by km12321** [bokeh/bokeh#2013]
Is it possible to set width and height of MultiSelect Widget? I tried creating a tall VBox and VBoxForm and having its only child be a MultiSelect but this did not affect the widget's dimensions. 

Also, when I change the value/options attributes of a MultiSelect widget programmatically, the widget gets populated with empty strings. The workaround I've been using is to re-render the container VBox, but it would be great if re-rendering could be avoided.

Thanks!

**Comment by km12321** [comment]
In general, is it possible to re-render certain nested containers/widgets or only top-level ones?

For example, assuming I am in a subclass of HBox:
self.widget1 = MultiSelect...
self.widget2 = MultiSelect...
self.box1 = VBox(children=[self.widget1, self.widget2])
self.children = [self.plots, self.box1]

After this is initially set up, if I want to re-render, say, only widget1, how would I go about doing this?
I tried: self.children[1].children[0] = MultiSelect... but this did not work.
However, if I recreate box1 and then do self.children[1] = self.box1, this does re-render both widget1 and widget2. Let me know if this makes sense

Thanks for your help!

**Comment by birdsarah** [comment]
Hi. I understand why you'd want to do it programatically, but there is a workaround. 

(sorry if I'm stating the obvious)

The multiwidget box is just an html element. Bokeh comes with its own css so you just need to override it from:

````css
select[multiple], select[size] {
  height: auto;
}
````
to 
````css
select[multiple], select[size] {
  height: 100px;  /* or whatever you want */
}
````

Having just written all this, I've realized that I've got a whole setup that makes that easy for me to do. I will be documenting it and publishing the code so others can check it out ~midApril, but its not anywhere for you to look at at the moment.

Sorry if this is no help at all :crying_cat_face:
